jujugui/static/gui/src/app/app.js

```diff
@@ -1285,12 +1286,14 @@ YUI.add('juju-gui', function(Y) {
         };
       }
       const getUserName = () => {
-        const credentials = this.user.controller;
-        return credentials ? credentials.user : undefined;
+        return this.user.username;
       };
       const loginToController = controllerAPI.loginWithMacaroon.bind(
         controllerAPI, this.bakery);
       const charmstore = this.get('charmstore');
+      const isLoggedIn = () => {
```

Comment:
arrow fn's allow you to drop the `return` and curlies if you only have a single statement, up to you, just mentioning it.

```JavaScript
const isLoggedIn = () => this.controllerAPI.userIsAuthenticated;
```